parser/model/ddl.go

```diff
@@ -211,7 +211,13 @@ type DDLReorgMeta struct {
 	SQLMode       mysql.SQLMode                    `json:"sql_mode"`
 	Warnings      map[errors.ErrorID]*terror.Error `json:"warnings"`
 	WarningsCount map[errors.ErrorID]int64         `json:"warnings_count"`
-	Location      *time.Location                   `json:"time_location"`
+	Location      *TimeZone                        `json:"time_zone"`
```

Comment:
Where do you set it at?

Reply:
I will set it in the next PR（There are a lot of problems involved in this PR, so I will fix this problem first
）.

Reply:
But I don't know what you want to fix and don't know if it works.

Reply:
It describes the problems we need to deal with: “The `Location` in the job is empty after decoding”.  It was also written in the original PR of worker to be dealt with.

Reply:
I mean the problem from the user's perspective. 
“The Location in the job is empty after decoding”, Sounds like a problem, but what would happen? I'm still confused.
Besides, why it's empty after decoding? And Why using `Name` and `Offset` can fix it?

Reply:
Because the Location object's fields are all lowercase, there are no useful fields after decoding.

Reply:
Ok, I get it. Please add an issue for this PR.

Reply:
@wjhuang2016 Done.